Classes/Views/SelectableCell.swift

```diff
@@ -47,6 +57,9 @@ class SelectableCell: UICollectionViewCell {
 
     override func layoutSubviews() {
         super.layoutSubviews()
+        accessibilityLabel = contentView.subviews
+            .flatMap { $0.accessibilityLabel }
+            .reduce("", { "\($0 ?? "").\n\($1)" })
```

Comment:
Mind moving this to an override of `accessibilityLabel`? Then it only executes when necessary instead of every layout pass.